src/Salt/UserBundle/Resources/views/user/index.html.twig

```diff
@@ -43,26 +43,31 @@
                     </td>
                     <td class="text-center">
                         <ul class="list-inline">
-                            <li>
-                                <a class="btn btn-primary" href="{{ path('admin_user_show', { 'id': user.id }) }}">show</a>
-                            </li>
-                            {% if not user.suspended %}
                                 <li>
-                                    <a class="btn btn-default" href="{{ path('admin_user_edit', { 'id': user.id }) }}">edit</a>
+                                    <a class="btn btn-primary" href="{{ path('admin_user_show', { 'id': user.id }) }}">show</a>
                                 </li>
-                            {% endif %}
-                            <li>
-                                {% if user.suspended %}
-                                    <a class="btn btn-default" href="{{ path('admin_user_activate', { 'id': user.id }) }}">Unsuspend</a>
-                                {% else %}
-                                    <a class="btn btn-default" href="{{ path('admin_user_suspend', { 'id': user.id }) }}">Suspend</a>
+                                {% if not user.suspended %}
+                                    <li>
+                                        <a class="btn btn-default" href="{{ path('admin_user_edit', { 'id': user.id }) }}">edit</a>
+                                    </li>
                                 {% endif %}
-                            </li>
-                            {% if user.pending %}
                                 <li>
-                                    <a class="btn btn-default" href="{{ path('admin_user_activate', { 'id': user.id }) }}">Approve</a>
+                                    {% if user.suspended %}
+                                        <a class="btn btn-default" href="{{ path('admin_user_activate', { 'id': user.id }) }}">Unsuspend</a>
+                                    {% else %}
+                                        <a class="btn btn-default" href="{{ path('admin_user_suspend', { 'id': user.id }) }}">Suspend</a>
+                                    {% endif %}
                                 </li>
-                            {% endif %}
+                                {% if user.pending %}
+                                    <li>
+                                        <a class="btn btn-primary" href="{{ path('admin_user_activate', { 'id': user.id }) }}">Approve</a>
+                                    </li>
+                                    {% if is_granted('ROLE_SUPER_USER') %} 
```

Comment:
Does it make sense that only a user with the ROLE_SUPER_USER role can `Reject` a user but other can `Approve`, especially since others can also `Suspend` the user by the above logic (which does the same thing as `Reject` does)?